src/moepkg/syntax/highlite.nim

```diff
@@ -229,19 +229,20 @@ proc isKeyword*(x: openArray[string], y: string): int =
 
 import syntaxc, syntaxcpp, syntaxcsharp, syntaxhaskell, syntaxjava,
        syntaxjavascript, syntaxmarkdown, syntaxnim, syntaxpython, syntaxrust,
-       syntaxyaml
+       syntaxshell, syntaxyaml
 proc getNextToken*(g: var GeneralTokenizer, lang: SourceLanguage) =
   case lang
-  of langC: cNextToken(g)
-  of langCpp: cppNextToken(g)
-  of langCsharp: csharpNextToken(g)
-  of langHaskell: haskellNextToken(g)
-  of langJava: javaNextToken(g)
-  of langJavaScript: javaScriptNextToken(g)
-  of langMarkdown: markdownNextToken(g)
-  of langNim: nimNextToken(g)
+  of langC: g.cNextToken
+  of langCpp: g.cppNextToken
+  of langCsharp: g.csharpNextToken
+  of langHaskell: g.haskellNextToken
+  of langJava: g.javaNextToken
+  of langJavaScript: g.javaScriptNextToken
+  of langMarkdown: g.markdownNextToken
+  of langNim: g.nimNextToken
   of langNone: assert false
-  of langPython: pythonNextToken(g)
-  of langRust: rustNextToken(g)
-  of langYaml: yamlNextToken(g)
+  of langPython: g.pythonNextToken
+  of langRust: g.rustNextToken
+  of langShell: g.shellNextToken
+  of langYaml: g.yamlNextToken
   else: discard
```

Comment:
@fox0430

The compiler considers this `else` unreachable as all languages are redirected appropriately.  Before making changes to this section, I would like to discuss possible options in order to avoid the introduction of bugs.  Maybe the assertion in line 243 could be omitted, for instance?

Reply:
@kevinmatthes 

I think might as well omit that one. Can you omit it?

Reply:
@fox0430

Okay, I will remove line 243. :+1: